Lib/fontTools/misc/psCharStrings.py

```diff
@@ -983,6 +983,16 @@ def compile(self, isCFF2=False):
 			return
 		opcodes = self.opcodes
 		program = self.program
+
+		if isCFF2:
+			# If present, remove return and endchar operators.
+			if program and program[-1] in ("return", "endchar"):
+				program = program[:-1]
```

Comment:
`del program[-1]` would avoid the copy

Reply:
Yeah, but I thought it was nicer to not alter self.program. But I can change it if you want.

Reply:
ah, you mean, in case the reference to `program` list was shared by some other object.. Ok, let's leave it like that.